#!/usr/bin/env python
PACKAGE = "synchronizer_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("LED_Mode",          bool_t,   0, "False: flash always; True: flash with cam",  False)
gen.add("LED_Brightness",    int_t,    0, "LED brightness", 50,  0, 180)

exit(gen.generate(PACKAGE, "synchronizer_ros", "Led"))